drivers/sensor/lsm6dsl/lsm6dsl_spi.c

```diff
@@ -0,0 +1,106 @@
+/* lsm6dsl_spi.c - SPI routines for LSM6DSL driver
+ */
+
+/*
+ * Copyright (c) 2018 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <spi.h>
+#include "lsm6dsl.h"
+
+#define LSM6DSL_SPI_READ		(1 << 7)
+
+#define SPI_CS NULL
+
+static struct spi_config lsm6dsl_spi_conf = {
+	.frequency = 16000000,
+	.operation = SPI_OP_MODE_MASTER | SPI_MODE_CPOL |
+	SPI_MODE_CPHA | SPI_WORD_SET(8) | SPI_LINES_SINGLE,
```

Comment:
indentation seems a bit wrong. Check your editor if it can apply for you the linux code style, so it will do the indentation properly etc...

Reply:
some of the coding style thing are already set in my vim.
I'll change this manually.